modules/packages/VisualDebug.chpl

```diff
@@ -110,6 +110,7 @@ private proc VDebugTree (what: vis_op, name: string, time: real, tagno: int,
 }
 
 
+ private var Vdebugstarted = false;
```

Comment:
Does this need to be atomic in case it's called from multiple tasks/locales?

Reply:
That would make it even more bulletproof, yeah.

Reply:
Implemented, if you could take another look.